msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -182,29 +183,9 @@
     <None Update="Platforms\net45\WindowsFormsWebAuthenticationDialogBase.cs" SubType="Form" />
     <None Update="Platforms\net45\SilentWindowsFormsAuthenticationDialog.cs" SubType="Form" />
   </ItemGroup>
-
-  <!-- Copies all artifacts needed to generate the nuget package. This should be replaced with msbuild /t:pack, however
-  bug https://github.com/NuGet/Home/issues/3891 blocks this, espcially that packaging rebuilds the Core assemblies, so we cannot sign it.-->
-  <Target Name="LayoutNuGet">
-    <PropertyGroup>
-      <NugetPackageRootFolder Condition="'$(NugetPackageRootFolder)' == ''">$([System.IO.Path]::Combine($(MSBuildThisFileDirectory), "..",".nugetlayout"))</NugetPackageRootFolder>
-      <NuspecFilePath>$([System.IO.Path]::Combine($(MSBuildThisFileDirectory), "..", "..", "build", "Microsoft.Identity.Client.nuspec"))</NuspecFilePath>
-    </PropertyGroup>
-    <ItemGroup>
-      <FilesToPack Include="bin\$(Configuration)\**\Microsoft.Identity.Client.dll" />
-      <FilesToPack Include="bin\$(Configuration)\**\Microsoft.Identity.Client.xml" />
-      <FilesToPack Include="bin\$(Configuration)\**\Microsoft.Identity.Client.pdb" />
-    </ItemGroup>
-
-    <RemoveDir Directories="$(NugetPackageRootFolder)" />
-    <Copy SourceFiles="@(FilesToPack)" DestinationFiles="$(NugetPackageRootFolder)\lib\%(RecursiveDir)%(FileName)%(Extension)" />
-    <Copy SourceFiles="$(NuspecFilePath)" DestinationFolder="$(NugetPackageRootFolder)" />
-  </Target>
-
-  <PropertyGroup>
-    <CodeAnalysisRuleSet>../../../core/ConfigureAwait.ruleset</CodeAnalysisRuleSet>
-  </PropertyGroup>
+   
   <ItemGroup>
     <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="All" />
```

Comment:
><PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="All" /> [](start = 4, length = 95)

do we then still need this one? It's not part of the new analyzers?

Reply:
We use 2 rules from AsyncUsageAnalyzers - 1. ensure ConfigureAwait is explicitly set and 2. Use Async suffix. 

The new analyzers have support for //2 but not for //1 so I decided to keep the AsyncUsageAnalyzers